source/installing.rst

```diff
@@ -44,6 +44,22 @@ Install the |percona-platform-postgresql-11| package using |apt-install|.
 
    $ sudo apt install percona-postgresql-11
 
+Note that this package will not install the components. To install these
+components use the appropriate packages:
+
+.. code-block:: bash
+
+   $ # To install pg_repack
+   $ sudo apt install percona-postgresql-11-repack
```

Comment:
maybe it would be better to use apt-get as sometimes(at least it was on ubuntu16) apt might ask to use apt-get instead